util/local_verdaccio_publish

```diff
@@ -35,21 +41,33 @@ const args = [
   '--no-git-reset',
   '--preid', 'ci',
   '--no-verify-access',
-  '--force-publish'
+  '--force-publish',
+  '--loglevel', 'warn',
+  '--no-progress'
 ]
-spawn('npx', args, pipeStdIo)
-  .on('close', (code) => {
+timeout = 60000 * 2
+console.log(`Starting lerna publish with timeout of ${timeout}`);
+spawn('npx', args, {
+    stdio: [process.stdin, process.stdout, process.stderr],
+    timeout: timeout
+}).on('close', (code, signal) => {
+    console.log(`lerna terminated due to receipt of signal ${signal} or code ${code}`);
     // The above command will make some modifications,
     // Roll them back
     // Ideally, we would find a way to not have to do this
     execSync('git checkout -- modules/**/package.json')
     execSync('git checkout -- lerna.json')
+    execSync('git restore package-lock.json')
 
     // Kill the background verdaccio server
-    verdaccio.kill()
-
+    verdaccioKilledStatus = verdaccio.kill()
+    console.log(`killing Verdaccio returned ${verdaccioKilledStatus}`);
+    
     // If this command had an error,
     // we need to forward this.
     // Otherwise the entire CI build may think that things succeeded.
     if (code !== 0) throw Error(`Exit code: ${code}`)
-  })
\ No newline at end of file
+
+    process.exit()
```

Comment:
My guess is that this is required because you have added a `timeout`.
That timeout is likely implemented with `setTimeout`.
This means when this is executed there is an event in the background.

Reply:
Hmm... it was hanging before I added the timeout. 
But I guess I could check to see if just exit fixes the hang, with out the time out.